Sources_v3/Workers/Background/MessageSender.swift

```diff
@@ -36,19 +36,20 @@ class MessageSender<ExtraData: ExtraDataTypes>: Worker {
     override init(database: DatabaseContainer, webSocketClient: WebSocketClient, apiClient: APIClient) {
         super.init(database: database, webSocketClient: webSocketClient, apiClient: apiClient)
 
-        // The observe can be set up on the background queue
-        DispatchQueue.global().async {
-            self.observer.onChange = { [unowned self] in self.handleChanges(changes: $0) }
+        // The observer can be set up on the background queue
+        sendingDispatchQueue.async { [weak self] in
+            self?.observer.onChange = { self?.handleChanges(changes: $0) }
             do {
-                try self.observer.startObserving()
+                try self?.observer.startObserving()
```

Comment:
Why do we start observing twice? 🤔

Reply:
I have no idea 🤷‍♂️

Reply:
fixed in 083437465d99712d179ecd2d0cc08975e13f859d